tests/paz/pipelines/human_pose_3d_tests.py

```diff
@@ -1,26 +1,50 @@
 import pytest
-from pipeline import SimpleBaselines
+from paz.pipelines.keypoints import SimpleBaselines
 import numpy as np
+import os
 from paz.backend.image import load_image
 from paz.backend.camera import Camera
-from linear_model import Simple_Baseline
-from keypoints_processors import SolveTranslation3D
+from paz.models.keypoint.simplebaselines import SimpleBaseline
 from scipy.optimize import least_squares
+from tensorflow.keras.utils import get_file
+from paz.backend.keypoints import filter_keypoints3D
+from paz.backend.keypoints import initialize_translation, solve_least_squares,\
+    get_bones_length, compute_reprojection_error,\
+    compute_optimized_pose3D
```

Comment:
try to follow the convention of standard imports, then library import and then our library imports.